ssh/server.go

```diff
@@ -121,6 +115,12 @@ type ServerConfig struct {
 	// attempts.
 	AuthLogCallback func(conn ConnMetadata, method string, err error)
 
+	// NextAuthMethodCallback, if non-nil, is called whenever an authentication
+	// method fails. It's called after AuthLogCallback, if set. The return
+	// values are the SSH auth types to suggest that the client try next. If
+	// empty, authentication fails.
+	NextAuthMethodCallback func(conn ConnMetadata, prevErrors []error) []string
```

Comment:
it must contain the `none` or equivalent auth methods.

Reply:
It doesn't contain none

Reply:
updated docs w/ URL